tools/dockerfile/grpc_artifact_python_manylinux_2_24_aarch64/Dockerfile

```diff
@@ -12,18 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The aarch64 wheels are being crosscompiled to allow running the build
-# on x64 machine. The dockcross/manylinux2014-aarch64 image is a x86_64
-# image with crosscompilation toolchain installed.
-# We use an older version of dockcross image that has gcc4.9.4 because it was built
-# before https://github.com/dockcross/dockcross/pull/449
-# Thanks to that, wheel build with this image aren't actually
-# compliant with manylinux2014, but only with manylinux_2_24
 FROM dockcross/manylinux2014-aarch64:20210826-19322ba
 
-# Make the grpc wheels correctly tagged with manylinux_2_24
-# instead of the incorrect manylinux2014 (see the note above).
-ENV AUDITWHEEL_PLAT manylinux_2_24_$AUDITWHEEL_ARCH
+# manylinux_2_17 is the preferred alias of manylinux2014
+ENV AUDITWHEEL_PLAT manylinux_2_17_$AUDITWHEEL_ARCH
```

Comment:
if this works, don't forget to rename the dockerfile to back to manylinux2014 (or to manylinux_2_17, but note that we are using manylinux2014 in the name of other docker images, so that might be a better choice)

Reply:
we also need to check the resulting wheel with auditwheel on aarch64 to make sure it's really manylinux_2_17 compliant (it is, according to my earlier experiment, but it's better to doublecheck).